code/game/objects/items/implants/implantcase.dm

```diff
@@ -10,51 +13,46 @@
 	throw_range = 5
 	w_class = WEIGHT_CLASS_TINY
 	custom_materials = list(/datum/material/glass=500)
+	///the implant within the case
 	var/obj/item/implant/imp = null
+	///Type of implant this will spawn as imp upon being spawned
 	var/imp_type
 
 
 /obj/item/implantcase/update_icon_state()
 	icon_state = "implantcase-[imp ? imp.implant_color : 0]"
 	return ..()
 
-
-/obj/item/implantcase/attackby(obj/item/W, mob/user, params)
-	if(istype(W, /obj/item/pen))
+/obj/item/implantcase/attackby(obj/item/I, mob/living/user, params)
```

Comment:
change I to something more descriptive please

Reply:
done